src/pages/learn/index.tsx

```diff
@@ -0,0 +1,266 @@
+                it accounts for the duration of credit accounts, and the age of
+                the oldest account. It also considers the percentage of
+                available credit used, the cumulative credit limit, and how
+                often interest was charged on your credit cards.
+              </p>
+            </div>
+            <div>
+              {' '}
+              <div className="flex items-center">
+                <div className="w-3 h-3 bg-rose-400 rounded-full mr-3" />
+                <h4 className="text-base font-bold">Velocity (40%)</h4>
+              </div>
+              <p className="text-lightgray text-xs mt-5">
+                It analyzes how many and how fast transactions occur in checking
+                account(s). It calculates the number and volume of monthly
+                recurring deposits and withdrawals, the cumulative monthly
+                income, and expenses.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className="bg-black p-10 mt-10 mb-20 rounded-xl">
+        <div className="flex items-baseline">
+          <img
+            src="./images/coinbaseLogo.svg"
+            alt="coinbase-logo"
+            className="w-32"
+          />
+          <h3 className="text-lg md:text-xl ml-3">Model</h3>
+        </div>
+        <p className="text-lightgray mt-5">
+          Your Coinbase data is grouped into 4 categories:{' '}
+          <span className="font-bold">KYC, history, liquidity, activity.</span>{' '}
+          The percentages in the donut chart reflect how important each of the
+          categories is in determining the SCRTSibyl score. The importance of
+          categories is fixed for the Coinbase model.
+        </p>
+
+        <div className="grid grid-rows-3 md:grid-rows-none md:grid-cols-4 gap-2 mt-10">
+          <div className="flex flex-col justify-evenly">
+            <div>
+              {' '}
+              <div className="flex items-center">
+                <div className="w-3 h-3 bg-white rounded-full mr-3" />
+                <h4 className="text-base font-bold">KYC (10%)</h4>
+              </div>
+              <p className="text-lightgray text-xs mt-5">
+                It checks whether you correctly kyc&apos;ed into Coinbase and
+                whether you own any active and non-zero balance accounts.
+              </p>
+            </div>
+            <div>
+              {' '}
+              <div className="flex items-center">
+                <div className="w-3 h-3 bg-blue-200 rounded-full mr-3" />
+                <h4 className="text-base font-bold">History (10%)</h4>
+              </div>
+              <p className="text-lightgray text-xs mt-5">
+                It looks at the length of transaction history and the age of
+                your longest-standing wallet.
+              </p>
+            </div>
+          </div>
+
+          <div className="col-span-2">
+            {' '}
```

Comment:
is there a purpose for these spaces everywhere?

Reply:
i think they're auto-created! Will remove them